python/chemiscope/structures/__init__.py

```diff
@@ -20,6 +20,19 @@
     ase_tensors_to_ellipsoids,
     ase_vectors_to_arrows,
 )
+from ._stk import (
+    _stk_valid_structures,
+    _stk_to_json,
+    convert_stk_bonds_as_shapes,
+    _stk_all_atomic_environments,
+    _stk_composition_properties,
+    _stk_list_atom_properties,
+    _stk_list_structure_properties,
+)
+
+__all__ = [
+    "convert_stk_bonds_as_shapes",
+]
```

Comment:
I don't think this is correct. `__all__` controls what gets imported when someone does `from xxx import *`. 

If this is a workaround the linter about unused import, feel free to use `# noqa` for this import!

Reply:
Done.